commons-packet/commons-packet-service/pom.xml

```diff
@@ -30,14 +30,14 @@
         <kernel.keygenerator.bouncycastle.version>1.3.0-SNAPSHOT</kernel.keygenerator.bouncycastle.version>
```

Comment:
Are we using this:
<kernel.keygenerator.bouncycastle.version>1.3.0-SNAPSHOT</kernel.keygenerator.bouncycastle.version>